radixconfig.yaml

```diff
@@ -29,6 +29,8 @@ spec:
           replicas: 2
         - environment: qa
           runAsNonRoot: true
+      variables:
+        CLUSTER_TYPE_EGRESS_IPS: undefined # Comma separated list of IPs of all Public IP Prefixes assigned to the ClusterType (production, playground, development)
```

Comment:
Why is "TYPE" in the name? Should it perhaps just be "CLUSTER_EGRESS_IPS"?

Reply:
fixed